pages/zap.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Using OWASP ZAP
+permalink: /zap/
+---
+
+# The OWASP Zed Attack Proxy (ZAP)
+
+[OWASP ZAP](https://www.owasp.org/index.php/OWASP_Zed_Attack_Proxy_Project) is a tool for doing automated vulnerability scanning.
+
+## Usage
+
+You will need a running application to test. If testing a live URL, make sure to let your team know, as they may see a spike in requests and errors.
+
+1. [Install the ZAP desktop app](https://github.com/zaproxy/zaproxy/wiki/Downloads).
+1. Start your application.
+    * If using Rails on Mac, start it with `rails server --binding 127.0.0.1` (presumably due to [this change](http://guides.rubyonrails.org/4_2_release_notes.html#default-host-for-rails-server)).
```

Comment:
When running a Rails 4.2 app from your local machine, `rails s` is sufficient to be able to visit the app in your browser at `localhost` or `127.0.0.1`. If you want to run the app from a virtual machine, such as a Vagrant box, and then visit the app on your local machine, you will need to use `rails s -b 0.0.0.0`. The binding option is also required if you want to be able to visit the app on your mobile device using your local machine's WiFi IP address.